diff --git a/node_modules/@nozbe/with-observables/dist/with-observables.cjs.js b/node_modules/@nozbe/with-observables/dist/with-observables.cjs.js
index 1d31a76..747c9e7 100644
--- a/node_modules/@nozbe/with-observables/dist/with-observables.cjs.js
+++ b/node_modules/@nozbe/with-observables/dist/with-observables.cjs.js
@@ -4,9 +4,9 @@ function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'defau
 
 var react = require('react');
 var hoistNonReactStatic = _interopDefault(require('hoist-non-react-statics'));
-var from = require('rxjs/observable/from');
-var combineLatest = require('rxjs/observable/combineLatest');
-var map = require('rxjs/operators/map');
+var from = require('rxjs/internal/observable/from');
+var combineLatest = require('rxjs/internal/observable/combineLatest');
+var map = require('rxjs/internal/operators/map');
 
 function _classCallCheck(instance, Constructor) {
   if (!(instance instanceof Constructor)) {
diff --git a/node_modules/@nozbe/with-observables/dist/with-observables.esm.js b/node_modules/@nozbe/with-observables/dist/with-observables.esm.js
index c4506d7..85f5307 100644
--- a/node_modules/@nozbe/with-observables/dist/with-observables.esm.js
+++ b/node_modules/@nozbe/with-observables/dist/with-observables.esm.js
@@ -1,8 +1,8 @@
 import { createElement, Component } from 'react';
 import hoistNonReactStatic from 'hoist-non-react-statics';
-import { from } from 'rxjs/observable/from';
-import { combineLatest } from 'rxjs/observable/combineLatest';
-import { map } from 'rxjs/operators/map';
+import { from } from 'rxjs/internal/observable/from';
+import { combineLatest } from 'rxjs/internal/observable/combineLatest';
+import { map } from 'rxjs/internal/operators/map';
 
 function _classCallCheck(instance, Constructor) {
   if (!(instance instanceof Constructor)) {
