diff --git a/node_modules/@shopify/restyle/dist/hooks/useDimensions.js b/node_modules/@shopify/restyle/dist/hooks/useDimensions.js
index 21f5e8a..8b2d096 100644
--- a/node_modules/@shopify/restyle/dist/hooks/useDimensions.js
+++ b/node_modules/@shopify/restyle/dist/hooks/useDimensions.js
@@ -9,9 +9,9 @@ var useDimensions = function () {
         setDimensions(window);
     };
     react_1.useEffect(function () {
-        react_native_1.Dimensions.addEventListener('change', onChange);
-        return function () { return react_native_1.Dimensions.removeEventListener('change', onChange); };
-    }, []);
+        var subscription = react_native_1.Dimensions.addEventListener('change', onChange);
+        return function () { return subscription && subscription.remove ? subscription.remove() : react_native_1.Dimensions.removeEventListener('change', onChange); };
+     }, []);
     return dimensions;
 };
 exports.default = useDimensions;
