diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/Screen.java b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/Screen.java
index b64380d..c108663 100644
--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/Screen.java
+++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/Screen.java
@@ -73,7 +73,7 @@ public class Screen extends ViewGroup implements ReactPointerEventsView {
 
   @Override
   public PointerEvents getPointerEvents() {
-    return mTransitioning ? PointerEvents.NONE : PointerEvents.AUTO;
+    return PointerEvents.AUTO;
   }
 
   @Override
