diff --git a/node_modules/react-native-web/dist/exports/Image/index.js b/node_modules/react-native-web/dist/exports/Image/index.js
index 8281e36..f106aff 100644
--- a/node_modules/react-native-web/dist/exports/Image/index.js
+++ b/node_modules/react-native-web/dist/exports/Image/index.js
@@ -216,7 +216,7 @@ function (_React$Component) {
           onLoad = _this$props2.onLoad,
           source = _this$props2.source;
       var event = {
-        nativeEvent: e
+        nativeEvent: {...e, width: _this._imageRef.naturalWidth, height: _this._imageRef.naturalHeight}
       };
       ImageUriCache.add(resolveAssetUri(source));
 
