patch-package
new file mode 120000
--- /dev/null
+++ b/node_modules/react-native/.bin/react-native
@@ -0,0 +1 @@
+../local-cli/wrong-react-native.js
\ No newline at end of file
--- a/node_modules/react-native/Libraries/StyleSheet/StyleSheet.js
+++ b/node_modules/react-native/Libraries/StyleSheet/StyleSheet.js
@@ -339,7 +339,7 @@ module.exports = {
   ) {
     let value;
 
-    if (typeof ReactNativeStyleAttributes[property] === 'string') {
+    if (ReactNativeStyleAttributes[property] === true) {
       value = {};
     } else if (typeof ReactNativeStyleAttributes[property] === 'object') {
       value = ReactNativeStyleAttributes[property];
