diff --git a/node_modules/react-native-camera/src/RNCamera.web.js b/node_modules/react-native-camera/src/RNCamera.web.js
new file mode 100644
index 0000000..4e36fc0
--- /dev/null
+++ b/node_modules/react-native-camera/src/RNCamera.web.js
@@ -0,0 +1,38 @@
+import React, { useRef, useCallback, useEffect } from 'react';
+import Quagga from 'quagga'
+
+function RNCamera({ style, onBarCodeRead }) {
+  const onDetected = useCallback(result => result?.codeResult && onBarCodeRead({data: result.codeResult.code}), [onBarCodeRead]);
+  const ref = useRef();
+  useEffect(() => {
+    Quagga.init({
+      inputStream: {
+        type:"LiveStream",
+        constraints: {width:640,height:480,facingMode:"environment"},
+        target: ref.current
+      },
+      locator: {patchSize:"medium",halfSample:true},
+      numOfWorkers: 2,
+      frequency: 10,
+      decoder: {readers:["ean_reader"]},
+      locate:true
+    }, function(err) {
+      if (err) {
+          return console.log(err);
+      }
+      Quagga.start();
+  });
+
+  Quagga.onDetected(onDetected);
+
+  return () => {
+    Quagga.offDetected(onDetected);
+    Quagga.stop();
+  }}, []);
+
+  return <div id="interactive" className='viewport' ref={ref}/>
+};
+
+RNCamera.Constants = {Type: {}}
+
+export default RNCamera;
