diff --git a/node_modules/react-native-fast-image/dist/react-native-fast-image.esm.js b/node_modules/react-native-fast-image/dist/react-native-fast-image.esm.js
index 77bd06e..6214842 100644
--- a/node_modules/react-native-fast-image/dist/react-native-fast-image.esm.js
+++ b/node_modules/react-native-fast-image/dist/react-native-fast-image.esm.js
@@ -53,9 +53,9 @@ function FastImageBase(_ref) {
       forwardedRef = _ref.forwardedRef,
       props = _objectWithoutPropertiesLoose(_ref, ["source", "tintColor", "onLoadStart", "onProgress", "onLoad", "onError", "onLoadEnd", "style", "fallback", "children", "resizeMode", "forwardedRef"]);
 
-  var resolvedSource = Image.resolveAssetSource(source);
+  var resolvedSource = FastImageViewNativeModule ? Image.resolveAssetSource(source) : source;
 
-  if (fallback) {
+  if (fallback || !FastImageViewNativeModule) {
     return React.createElement(View, {
       style: [styles.imageContainer, style],
       ref: forwardedRef
@@ -100,7 +100,7 @@ FastImage.cacheControl = cacheControl;
 FastImage.priority = priority;
 
 FastImage.preload = function (sources) {
-  return FastImageViewNativeModule.preload(sources);
+  return FastImageViewNativeModule && FastImageViewNativeModule.preload(sources);
 };
 
 var styles = /*#__PURE__*/StyleSheet.create({
@@ -109,7 +109,7 @@ var styles = /*#__PURE__*/StyleSheet.create({
   }
 }); // Types of requireNativeComponent are not correct.
 
-var FastImageView = /*#__PURE__*/requireNativeComponent('FastImageView', FastImage, {
+var FastImageView = FastImageViewNativeModule && /*#__PURE__*/requireNativeComponent('FastImageView', FastImage, {
   nativeOnly: {
     onFastImageLoadStart: true,
     onFastImageProgress: true,
