diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index 6acd02d..5e03b1b 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -136,8 +136,12 @@ afterEvaluate {
             if (enableHermes) {
                 doLast {
                     def hermesFlags;
+                    ant.move(
+                        file: jsBundleFile,
+                        tofile: "${jsBundleFile}_temp"
+                    );
                     exec {
-                        if (targetName.toLowerCase().contains("release")) {
+                        if (!targetName.toLowerCase().contains("debug")) {
                             // Can't use ?: since that will also substitute valid empty lists
                             hermesFlags = config.hermesFlagsRelease
                             if (hermesFlags == null) hermesFlags = ["-O", "-output-source-map"]
@@ -145,8 +149,11 @@ afterEvaluate {
                             hermesFlags = config.hermesFlagsDebug
                             if (hermesFlags == null) hermesFlags = []
                         }
-                        commandLine(getHermesCommand(), "-emit-binary", "-out", jsBundleFile, jsBundleFile, *hermesFlags)
+                        commandLine("cmd", "/c", getHermesCommand(), "-emit-binary", "-out", jsBundleFile, "${jsBundleFile}_temp", *hermesFlags)
                     }
+                    ant.delete(
+                        file: "${jsBundleFile}_temp"
+                    )
                     if (hermesFlags.contains("-output-source-map")) {
                         ant.move(
                             // Hermes will generate a source map with this exact name
