diff --git a/node_modules/react-native-calendars/src/agenda/index.js b/node_modules/react-native-calendars/src/agenda/index.js
index a0fb939..3eca8ec 100644
--- a/node_modules/react-native-calendars/src/agenda/index.js
+++ b/node_modules/react-native-calendars/src/agenda/index.js
@@ -16,7 +16,7 @@ const HEADER_HEIGHT = 104;
 const KNOB_HEIGHT = 24;
 // Fallback when RN version is < 0.44
 const {Text, View, Dimensions, Animated, ViewPropTypes} = ReactNative;
-const viewPropTypes = ViewPropTypes || View.propTypes;
+const viewPropTypes = ViewPropTypes || View.propTypes || {};
 
 /**
  * @description: Agenda component
diff --git a/node_modules/react-native-calendars/src/calendar/index.js b/node_modules/react-native-calendars/src/calendar/index.js
index d2a247c..28e6a42 100644
--- a/node_modules/react-native-calendars/src/calendar/index.js
+++ b/node_modules/react-native-calendars/src/calendar/index.js
@@ -19,7 +19,7 @@ import {SELECT_DATE_SLOT} from '../testIDs';
 
 //Fallback when RN version is < 0.44
 const {View, ViewPropTypes} = ReactNative;
-const viewPropTypes = ViewPropTypes || View.propTypes;
+const viewPropTypes = ViewPropTypes || View.propTypes || {};
 const EmptyArray = [];
 
 /**
diff --git a/node_modules/react-native-calendars/src/component-updater.js b/node_modules/react-native-calendars/src/component-updater.js
index 19402f0..f6f5c5e 100644
--- a/node_modules/react-native-calendars/src/component-updater.js
+++ b/node_modules/react-native-calendars/src/component-updater.js
@@ -1,8 +1,9 @@
-const _ = require('lodash');
+const isEqual = require('lodash/isEqual');
+const get = require('lodash/get');
 
 function shouldUpdate(a, b, paths) {
   for (let i = 0; i < paths.length; i++) {
-    const equals = _.isEqual(_.get(a, paths[i]), _.get(b, paths[i]));
+    const equals = isEqual(get(a, paths[i]), get(b, paths[i]));
     if (!equals) {
       return true;
     }
